RELEASE_NOTES.md

```diff
@@ -230,3 +230,61 @@ query { decidim { version } }
 This no longer returns the running Decidim version by default and instead it will result to `null` being reported as the version number.
 
 If you would like to re-enable exposing the Decidim version number through the GraphQL API, you may do so by setting the `DECIDIM_API_DISCLOSE_SYSTEM_VERSION` environment variable to `true`. However, this is highly discouraged but may be required for some automation or integrations.
+
+### 5.2. Changes in the routing
+
+As we were upgrading the application to Rails 7.1, we have noticed there are some changes in the routing system that led us to change the way participatory space mounting points are being used by Decidim.
+
+Previously, the participatory space routes were mounted like follows in either the Core or Admin.
+
+```ruby
+  Decidim.participatory_space_manifests.each do |manifest|
+    mount manifest.context(:admin).engine, at: "/", as: "decidim_admin_#{manifest.name}"
+  end
+```
+
+As of [\#13294](https://github.com/decidim/decidim/pull/13294), we have changed the way of mounting. Now, each one of the Participatory Spaces are being installed specifically from their own modules like follows:
+
+```ruby
+  initializer "decidim_assemblies.mount_routes" do
+    Decidim::Core::Engine.routes do
+      mount Decidim::Assemblies::Engine, at: "/", as: "decidim_assemblies"
+    end
+  end
+```
+
+As a developer or as a user, you should not feel the difference of this change, however, we would like to emphasize this change, as the upgrade may impact other community libraries.
+
```

Comment:
Depending on the kind of developer/implementer this could bit them. I changed the wording of the phrase and keep the spirit in my other comment, let me know if you like it. 

```suggestion
```